### PR TITLE
Image metadata CI check

### PR DESCRIPTION
**Pull request by doofus-01**
I don't know how to test this directly in the PR, but I did test in a different repository.

- It only tests for Artist and Copyright tags, I guess the other tags can be optional.
- It doesn't sift through all images in the repository, it just looks at images in a PR.

### CODE REVIEW

File: .github/workflows/image_metadata.yml [hunk 1]
```diff
@@ -0,0 +1,67 @@
+name: Image metadata check CI
+
+on:
+  #push:
+  #  branches: [master]
+  pull_request:
+    paths:
+      - '**.webp'
+      - '**.png'
+      - '**.jpg'
```
Comment (Pentarctagon) [inline .github/workflows/image_metadata.yml:10]: It'd be good to check for .jpeg as well.

**Comment by doofus-01** [comment]
I will disable the current CSV check in a separate commit.

**Comment by Pentarctagon** [comment]
If it's no longer used and up to date then it should be removed - it'll always be in the git history if someone wants it.

File: .github/workflows/image_metadata.yml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        run: |
+          sudo apt-get install --assume-yes exiftool
+
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: read/export image metadata
+        run: |
+          mapfile -t image_files < <(git diff --name-only ${BASE_SHA} ${SHA} | grep '\.webp$')
```
Comment (soliton-) [inline .github/workflows/image_metadata.yml:37]: Why is this just looking for webp files?

File: .github/workflows/image_metadata.yml [hunk 3]
```diff
@@ -0,0 +1,68 @@
+name: Image metadata check CI
+
+on:
+  #push:
+  #  branches: [master]
```
Comment (soliton-) [inline .github/workflows/image_metadata.yml:5]: Why is this commented out?

**Comment by doofus-01** [comment]
I also need to fix the case of CC0 license.

File: .github/workflows/image_metadata.yml (hunk 2 above)
Comment (doofus-01) [reply .github/workflows/image_metadata.yml:37]: That was a test that shouldn't have gotten in there...

**Pull request merged**